homeassistant/components/media_player/mpd.py

```diff
@@ -225,11 +248,15 @@ def media_previous_track(self):
 
     def play_media(self, media_type, media_id, **kwargs):
         """Send the media player the command for playing a playlist."""
-        _LOGGER.info(str.format("Playing playlist: {0}", media_id))
-        if media_type == MEDIA_TYPE_PLAYLIST:
```

Comment:
OK, I see. I agree, if the playlist fails it should only fail once, so we should fix that problem at its source.